### PR TITLE
Adding multiple db connections support for django-instrumentation's sqlcommenter

### PR DESCRIPTION
**Pull request by Thiyagu55**
# Multiple DB Connection support for sqlcommenter django-instrumetation

Opentelemetry-instrumentation-django's sqlcommenter will now be able to support multiple database connections.

## Type of change

Please delete options that are not relevant.

- [X] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Unit test has been added named test_multiple_connection_support

- [X] test_multiple_connection_support

# Checklist:

- [X] Followed the style guidelines of this project
- [X] Changelogs have been updated
- [X] Unit tests have been added

### CODE REVIEW

**Comment by Thiyagu55** [comment]
@srikanthccv for your notice

File: instrumentation/opentelemetry-instrumentation-dbapi/src/opentelemetry/instrumentation/dbapi/__init__.py [hunk 1]
```diff
@@ -47,7 +47,7 @@
 from opentelemetry.instrumentation.dbapi.version import __version__
 from opentelemetry.instrumentation.utils import (
     _generate_opentelemetry_traceparent,
-    _generate_sql_comment,
+    generate_sql_comment,
```
Comment (srikanthccv) [inline instrumentation/opentelemetry-instrumentation-dbapi/src/opentelemetry/instrumentation/dbapi/__init__.py:50]: Why make these public?

**Review by srikanthccv — APPROVED** [review]
LGTM, Please make the sqlcommenter specific utils private.

File: instrumentation/opentelemetry-instrumentation-dbapi/src/opentelemetry/instrumentation/dbapi/__init__.py (hunk 1 above)
Comment (Thiyagu55) [reply instrumentation/opentelemetry-instrumentation-dbapi/src/opentelemetry/instrumentation/dbapi/__init__.py:50]: As we are importing the function across libraries I had changed it to private.

Comment (Thiyagu55) [reply instrumentation/opentelemetry-instrumentation-dbapi/src/opentelemetry/instrumentation/dbapi/__init__.py]: Reverted the changes

File: opentelemetry-instrumentation/src/opentelemetry/instrumentation/utils.py [hunk 2]
```diff
@@ -119,32 +128,30 @@ def _start_internal_or_server_span(
...
+            f"{url_quote(key)}={url_quote(value)!r}"
             for key, value in sorted(meta.items())
             if value is not None
         )
         + "*/"
     )
 
 
-def _url_quote(s):  # pylint: disable=invalid-name
+def url_quote(s):  # pylint: disable=invalid-name
```
Comment (srikanthccv) [inline opentelemetry-instrumentation/src/opentelemetry/instrumentation/utils.py:154]: make this also private

File: opentelemetry-instrumentation/src/opentelemetry/instrumentation/utils.py [hunk 3]
```diff
@@ -155,6 +162,17 @@ def _url_quote(s):  # pylint: disable=invalid-name
     return quoted.replace("%", "%%")
 
 
+def get_opentelemetry_values():
```
Comment (srikanthccv) [inline opentelemetry-instrumentation/src/opentelemetry/instrumentation/utils.py:165]: this as well

File: opentelemetry-instrumentation/src/opentelemetry/instrumentation/utils.py [hunk 4]
```diff
@@ -26,6 +26,15 @@
...
 from opentelemetry.trace import Span, StatusCode
 
+try:
+    from opentelemetry.trace.propagation.tracecontext import (
+        TraceContextTextMapPropagator,
+    )
+
+    propagator = TraceContextTextMapPropagator()
+except ImportError:
+    propagator = None
```
Comment (srikanthccv) [inline opentelemetry-instrumentation/src/opentelemetry/instrumentation/utils.py:36]:
```suggestion

from opentelemetry.trace.propagation.tracecontext import (
    TraceContextTextMapPropagator,
)

_propagator = TraceContextTextMapPropagator()
```

**Review by srikanthccv — CHANGES_REQUESTED** [review]
blocking to prevent accidental merge. @Thiyagu55 please make the functions private since they are not user facing.

**Review by srikanthccv — APPROVED** [review]

**Pull request merged**